.github/workflows/integration-tests.yml

```diff
@@ -20,6 +20,12 @@ jobs:
       with:
         go-version: 1.18
     - name: Install Flow CLI
-      run: bash -ci "$(curl -fsSL https://raw.githubusercontent.com/onflow/flow-cli/master/install.sh)" -- v1.3.2
```

Comment:
heads up, I always pin this because flow versions have broken things for me between minor version in the past. Totally fine to remove, just a heads up that it can cause issues

Reply:
Good callout. Just pinned to `v1.3.4`